app.js

```diff
@@ -19,6 +20,7 @@ app.use(morgan('tiny'));
 app.use(authorization);
 // We mount our routers.
 app.use('/', clientRouter);
+app.use('/', managerRouter);
```

Comment:
Let's discuss how to leave this URLs before wrapping up, might be the right time to go with `/manager` and `/client`